src/lists.rs

```diff
@@ -623,7 +623,10 @@ pub fn extract_post_comment(
         post_snippet
     };
 
-    if !post_snippet_trimmed.is_empty() {
+    let removed_newline_snippet = post_snippet_trimmed.trim();
+    if !post_snippet_trimmed.is_empty()
+        && (removed_newline_snippet.starts_with("//") || removed_newline_snippet.starts_with("/*"))
```

Comment:
Do we need this because `const` gets duplicated? If so, could you add a FIXME comment so that we don't forget to send a fix to the libsyntax?

Reply:
Yes. post_snippet includes const like this.

```
[src/lists.rs:625] post_snippet = ", const"
[src/lists.rs:625] post_snippet = ": usize, const"
```